### DISCUSSION TITLE
Aula - Listas Assíncronas

### DISCUSSION THREAD

**Discussion by silvarafaell** [balta-io/7187#35] [Perguntas e Respostas] [answered]
quando coloquei o *ngFor="let product of products$ | async" não reconheceu, mas se eu tirar do codigo dá certo. 
![erro no curso do balta](https://user-images.githubusercontent.com/22201401/126655905-23f7f2e8-89ef-485a-a125-ca5b0976d3a3.png)

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde @silvarafaell, foi um problema na sua API!
Consegue consultar ela por fora e ver se está respondendo a requisição no endpoint que você chamou no Angular?

**Comment by silvarafaell** [comment]
Bom dia @andrebaltieri, depois da sua dica sobre API eu vi que não estava em execução. Obrigado pela ajuda.
